lib/plaindatetime.ts

```diff
@@ -31,7 +31,7 @@ export class PlainDateTime implements Temporal.PlainDateTime {
     millisecond = 0,
     microsecond = 0,
     nanosecond = 0,
-    calendar = ES.GetISO8601Calendar()
+    calendar: Temporal.CalendarProtocol | string = ES.GetISO8601Calendar()
```

Comment:
Similarly, should this be reflected back into the types? `Temporal.PlainDateTime` doesn't accept a string here, just a `CalendarProtocol`, but there's definitely places it's passed a string in the implementation.

Reply:
Yes, it looks like the spec accepts strings in the constructor for PlainXXX (except PlainTime) and ZonedDateTime. So it should be OK to update the public types of those constructors.

Reply:
Thanks! I changed this both for PlainDateTime and PlainDate.